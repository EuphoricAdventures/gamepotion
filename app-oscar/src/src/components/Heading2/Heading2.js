import React from 'react'
import styled from 'styled-components'
import { fontAlternative, colours } from '../../styleAbstractions'

const StyledHeading2 = styled.h2`
  ${fontAlternative}
  font-size: 120%;
  color: ${colours.fore};
`

const Heading2 = ({ children }) => {
  return (
    <StyledHeading2 className='component--heading2'>
      {children}
    </StyledHeading2>
  )
}

export default Heading2
