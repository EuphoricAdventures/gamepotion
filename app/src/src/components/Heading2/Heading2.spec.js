import React from 'react'
import renderer from 'react-test-renderer'
import 'jest-styled-components'
import Heading2 from './Heading2'

it('renders', () => {
  const tree = renderer
    .create(
      <Heading2>Hello, World</Heading2>
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
