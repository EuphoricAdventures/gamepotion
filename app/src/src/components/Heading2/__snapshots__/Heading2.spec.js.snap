// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
.c0 {
  font-family: "Rubik",-apple-system,BlinkMacSystemFont,"Segoe UI",sans-serif;
  font-weight: 400;
  -webkit-text-size-adjust: 100%;
  text-size-adjust: 100%;
  -webkit-font-smoothing: antialiased;
  font-size: 120%;
  color: #2e3131;
}

<h2
  className="component--heading2 c0"
>
  Hello, World
</h2>
`;
